### PR TITLE
fix(chat): hide unpublish option from change path dialog (Issue #1501)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

hide unpublish option from change path dialog

Issues:

- #1501 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Common/FolderContextMenu.tsx [hunk 1]
```diff
@@ -146,7 +148,10 @@ export const FolderContextMenu = ({
         name: t('Unpublish'),
         dataQa: 'unpublish',
         display:
-          isPublishingEnabled && isItemPublic(folder.id) && !!onUnpublish,
+          isPublishingEnabled &&
+          isItemPublic(folder.id) &&
+          !!onUnpublish &&
+          isSidePanelFolder,
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/FolderContextMenu.tsx:154]: use `featureType !== FeatureType.File` instead

Comment (Alexander-Kezik) [reply apps/chat/src/components/Common/FolderContextMenu.tsx:154]: but we have `type === SharingType.ConversationFolder ? FeatureType.Chat : FeatureType.Prompt` in `ChangePtathDialog.tsx`

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — APPROVED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1507.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**